repos/fdbt-site/src/pages/pointToPointPeriodProduct.tsx

```diff
@@ -118,6 +127,12 @@ export const getServerSideProps = (ctx: NextPageContextWithSession): { props: Po
     const csrfToken = getCsrfToken(ctx);
     const operatorAttribute = getSessionAttribute(ctx.req, OPERATOR_ATTRIBUTE);
     const product = getSessionAttribute(ctx.req, POINT_TO_POINT_PRODUCT_ATTRIBUTE);
+    const fareTypeAttribute = getRequiredSessionAttribute(ctx.req, FARE_TYPE_ATTRIBUTE);
```

Comment:
If we use getFareTypeFromFromAttributes , we dont need to check to see if its defined.

Reply:
Hmm - unsure you can use the function in a serverSideProps as it takes the request as an argument